apps/web/src/views/Farms/FarmsV3.tsx

```diff
@@ -269,38 +267,24 @@ const Farms: React.FC<React.PropsWithChildren> = ({ children }) => {
 
   const archivedFarms = farmsLP
 
-  const stakedOnlyFarms = activeFarms.filter((farm) => {
-    if (farm.version === 3) {
-      return farm.stakedPositions.length > 0
-    }
-    return (
-      farm.userData &&
-      (new BigNumber(farm.userData.stakedBalance).isGreaterThan(0) ||
-        new BigNumber(farm.userData.proxy?.stakedBalance).isGreaterThan(0))
-    )
-  })
-
-  const stakedInactiveFarms = inactiveFarms.filter((farm) => {
-    if (farm.version === 3) {
-      return farm.stakedPositions.length > 0
-    }
-    return (
-      farm.userData &&
-      (new BigNumber(farm.userData.stakedBalance).isGreaterThan(0) ||
-        new BigNumber(farm.userData.proxy?.stakedBalance).isGreaterThan(0))
-    )
-  })
-
-  const stakedArchivedFarms = archivedFarms.filter((farm) => {
-    if (farm.version === 3) {
-      return farm.stakedPositions.length > 0
-    }
-    return (
-      farm.userData &&
-      (new BigNumber(farm.userData.stakedBalance).isGreaterThan(0) ||
-        new BigNumber(farm.userData.proxy?.stakedBalance).isGreaterThan(0))
-    )
-  })
+  const getStakedFarms = useCallback((farmsData) => {
```

Comment:
same here I think it's safe to move this function outside the component

Reply:
Done